docs/Operation-Architecture/Orbit_UI/File_Operations/Deleting_Files.rst

```diff
@@ -30,67 +33,39 @@ To delete files from a selected bucket:
 Deleting Versioned Objects
 --------------------------
 
-To completely delete a versioned object, you must issue S3 API commands
-from the command line.
-
-If you have not already done so, follow the instructions at “Zenko from
-the Command Line,” on page 1 to configure one of your nodes to accept
-AWS S3 CLI commands.
-
-From your server node,
-
-#. List object versions:
+Deleting versioned objects is difficult because cloud servers are biased towards
+preserving data. While this is useful, it can become problematic when large 
+numbers of objects are under management (during stress testing, for example).
 
-   ::
-
-     $ aws s3api list-object-versions --bucket <bucket> --endpoint-url http://<zenko.endpoint.url> --profile <aws-profile>
-
-   The ``--profile`` option is only required if you have set up more
-   than the default profile. Check this from your Zenko node with
-   ``cat ~/.aws/credentials``. If only the ``[default]`` profile is
-   returned, omit the ``--profile`` option.
-
-   The above command returns the object’s version ID and key.
-
-   ::
-
-     {
-       "DeleteMarkers": [
-          {
-             "Owner": {
-                "DisplayName": "user",
-                "ID": "e25bcaf4c8ed41aba50891a671e68d76eddb405bd620252075388c340925c77e"
-             },
-             "IsLatest": true,
-             "VersionId": "39383436323730383139313839343939393939395247303031202032302e30",
-             "Key": "file.txt",
-             "LastModified": "2018-09-18T17:30:08.103Z"
-          }
-       ]
-     }
+To completely delete a versioned object, you must issue S3 API commands
+from the command line.
 
-#. To delete object versions, use the command:
+If you have not already done so, follow the instructions at
+:ref:`Zenko from the Command Line` to configure one of your nodes to accept
+AWS S3 CLI commands.
 
-    ::
+Zenko provides command-line scripts that enable removing versioned objects 
+completely (both removing the object data and its object ID from the namespace).
 
-       $ aws s3api delete-object --bucket <bucket-name> --version-id <versionID> --key <key> --endpoint-url http://<zenko.endpoint.url> --profile <aws-profile>
+The cleanupBuckets.js script is available in the s3utils pod. 
 
-    which in the present example returns:
+Run it as follows:
 
-    ::
+  # Exec into the pod with ``kubectl exec -it s3utils bash``
```

Comment:
We don't enable this pod by default so before we exec into the pod, it has to be enable in either the options file or at run time.

This is what it would look like in the options file.
```
maintenance:
  debug:
    enabled: true
```

The other option is to have it something like doesn't require options file configuration and will simply instantiate and launch you into an s3utils pod.
`kubectl run s3utils --image=zenko/s3utils:0.5 -it bash`